Makefile

```diff
@@ -23,6 +25,7 @@ check-go-static:
 
 install:
 	$(QUIET_INST)install -D $(TARGET) $(DESTDIR)$(BINDIR)/$(TARGET) || exit 1;
+	$(QUIET_INST)install -D config/$(CONFIG) $(DESTDIR)$(SYSCONFDIR)/clear-containers/$(CONFIG) || exit 1;
```

Comment:
This was a thoughtless copy from previous line. We dont require it at all. Will take that out.